docs/ServerConfiguration.md

```diff
@@ -40,6 +40,8 @@ exoframeNetwork: 'exoframe'
 updateChannel: 'stable'
 
 # path to folder with authorized_keys, default "~/.ssh"
+# NOTE: this must match the path to the public key directory **inside the exoframe container**
```

Comment:
I'd clarify that with `When running in docker container...` since exoframe server can also run without docker container 🤔

Reply:
Ah. I didn't know that 🙂 

Then maybe this note should rather go into the ServerInstallation.md file, since this only seems to be a problem in the case you're running the server through Docker, no?

Reply:
That might indeed be a better place to document this 🤔